sdk/servicebus/service-bus/test/renewLock.spec.ts

```diff
@@ -118,216 +118,6 @@ describe("Unpartitioned Queue - Lock Renewal #RunInBrowser", function(): void {
   });
 });
 
-describe("Partitioned Queue - Lock Renewal", function(): void {
```

Comment:
There are 7 tests here, 4 are for manual lock renewal, 3 are for auto lock renewal. 
#6442 talks about removing the entity variations for the auto lock renewal tests only, not the manual lock renewal ones.
This is because auto lock renewal is a client side feature where we maintain timers and then invoke the lock renewal method.

Please retain the variations for the manual lock renewal tests as they cover service side reactions as well.